woodwork/tests/accessor/test_column_accessor.py

```diff
@@ -440,20 +445,28 @@ def test_series_methods_on_accessor_returning_series_invalid_schema(sample_serie
 
 
 def test_series_methods_on_accessor_other_returns(sample_series):
-    xfail_dask_and_koalas(sample_series)
+    xfail_koalas(sample_series)
     series = sample_series.astype('category')
     series.ww.init()
     col_shape = series.ww.shape
     series_shape = series.shape
+    if dd and isinstance(series, dd.Series):
```

Comment:
You could create a helper function
```python
def _is_dask_series(data):
    if dd and isinstance(data, dd.Series):
        return True
    return False
```

Reply:
Yeah, we've got these calls all throughout the code as well as a dataframe version (and will also have them for Koalas). Maybe we should clean this up in a separate issue?

Reply:
Sure, can go in the improvements epic

Reply:
Created #628  for this update